utils/azure/storage.py

```diff
@@ -25,3 +25,33 @@ def instantiate_blob_client(
     blob_service_client = BlobServiceClient(account_url, credential=sp_credential)
 
     return blob_service_client
+
+
+def get_unique_jobs_from_blobs(blob_list: list = []) -> list:
+    """Function to extract unique job IDs from a list of blobs.
+    Args:
+        blob_list (list): List of blobs from Azure Storage.
+    Returns:
+        list: List of unique job IDs.
+    """
+    unique_jobs = set()
+    for blob in blob_list:
```

Comment:
```suggestion
    unique_jobs = set()
    blob_list = blob_list if blob_list is not None else []
    for blob in blob_list:
```

Make it an empty list if it is `None`.